requirements.txt

```diff
@@ -1,16 +1,11 @@
 Pillow>=4.1.1
 redis>=2.10.5
-mockredispy>=2.9.3
 
 # blosc 1.7.0 fails intermittently in the lambda environment.  Pinning at
 # 1.5.0 for now.
 blosc==1.5.0
-nose2
 numpy>=1.11.1
 
-# 1.3.8 adds cfn-lint which has an dependency on aws-sam-translator 1.11.0
-# which breaks during installation with Python 3.5.
-moto==1.3.7
 boto3
 hvac==0.6.3 # Pinned due to is_initialized call to check vault status being deprecated. Check: https://hvac.readthedocs.io/en/stable/source/hvac_v1.html
```

Comment:
Aren't we merged with refactor now?

Reply:
No, the refactor PR has not been merged, which fixes this.

Looking through the repo the only reason for the `hvac` library is something to due to getting dependencies installed in the lambda environment (4baffc)